src/controllers/keyboard/keyboardeventfilter.cpp

```diff
@@ -127,9 +154,15 @@ bool KeyboardEventFilter::eventFilter(QObject*, QEvent* e) {
 }
 
 QKeySequence KeyboardEventFilter::getKeySeq(QKeyEvent* e) {
-    QString modseq;
     QKeySequence k;
 
+    if (e->key() >= 0x01000020 && e->key() <= 0x01000023) {
```

Comment:
```suggestion
    if (e->key() >= Qt::Key_Shift && e->key() <= Qt::Key_Alt) {
```
And how about Qt::Key_AltGr?

Reply:
adopted in #11526